Here's a first nixpkgs-review run:

Result of `nixpkgs-review pr 151782` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>octave-jit</li>
    <li>octavePackages.fem-fenics</li>
    <li>octavePackages.vibes</li>
    <li>octavePackages.vrml</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>fldigi</li>
  </ul>
</details>
<details>
  <summary>138 packages built:</summary>
  <ul>
    <li>afterstep</li>
    <li>alsa-tools</li>
    <li>bumblebee</li>
    <li>crowbar</li>
    <li>csound</li>
    <li>csound-qt</li>
    <li>cubicsdr</li>
    <li>dillo</li>
    <li>dump1090</li>
    <li>ensemble-chorus</li>
    <li>eureka-editor</li>
    <li>exrdisplay</li>
    <li>faust2csound</li>
    <li>flightgear</li>
    <li>fllog</li>
    <li>flmsg</li>
    <li>flpsed</li>
    <li>flrig</li>
    <li>fltk</li>
    <li>fltk-minimal</li>
    <li>fltk14</li>
    <li>fltk14-minimal</li>
    <li>fltrator</li>
    <li>flwrap</li>
    <li>gama</li>
    <li>giac-with-xcas</li>
    <li>giada</li>
    <li>gmsh</li>
    <li>gnss-sdr</li>
    <li>gnuradio3_8Packages.ais</li>
    <li>gnuradio3_8Packages.limesdr</li>
    <li>gnuradio3_8Packages.osmosdr</li>
    <li>gqrx</li>
    <li>gqrx-gr-audio</li>
    <li>gqrx-portaudio</li>
    <li>ja2-stracciatella</li>
    <li>jwm-settings-manager</li>
    <li>librsb</li>
    <li>limesuite</li>
    <li>liquidwar</li>
    <li>lmms</li>
    <li>minc_widgets</li>
    <li>mup</li>
    <li>new-session-manager</li>
    <li>octave</li>
    <li>octaveFull</li>
    <li>octavePackages.arduino</li>
    <li>octavePackages.audio</li>
    <li>octavePackages.bim</li>
    <li>octavePackages.bsltl</li>
    <li>octavePackages.cgi</li>
    <li>octavePackages.communications</li>
    <li>octavePackages.control</li>
    <li>octavePackages.data-smoothing</li>
    <li>octavePackages.database</li>
    <li>octavePackages.dataframe</li>
    <li>octavePackages.dicom</li>
    <li>octavePackages.divand</li>
    <li>octavePackages.doctest</li>
    <li>octavePackages.econometrics</li>
    <li>octavePackages.financial</li>
    <li>octavePackages.fits</li>
    <li>octavePackages.fpl</li>
    <li>octavePackages.fuzzy-logic-toolkit</li>
    <li>octavePackages.ga</li>
    <li>octavePackages.general</li>
    <li>octavePackages.generate_html</li>
    <li>octavePackages.geometry</li>
    <li>octavePackages.gsl</li>
    <li>octavePackages.image</li>
    <li>octavePackages.image-acquisition</li>
    <li>octavePackages.instrument-control</li>
    <li>octavePackages.interval</li>
    <li>octavePackages.io</li>
    <li>octavePackages.level-set</li>
    <li>octavePackages.linear-algebra</li>
    <li>octavePackages.lssa</li>
    <li>octavePackages.ltfat</li>
    <li>octavePackages.mapping</li>
    <li>octavePackages.matgeom</li>
    <li>octavePackages.miscellaneous</li>
    <li>octavePackages.msh</li>
    <li>octavePackages.mvn</li>
    <li>octavePackages.nan</li>
    <li>octavePackages.ncarray</li>
    <li>octavePackages.netcdf</li>
    <li>octavePackages.nurbs</li>
    <li>octavePackages.ocl</li>
    <li>octavePackages.octclip</li>
    <li>octavePackages.octproj</li>
    <li>octavePackages.optics</li>
    <li>octavePackages.optim</li>
    <li>octavePackages.optiminterp</li>
    <li>octavePackages.parallel</li>
    <li>octavePackages.quaternion</li>
    <li>octavePackages.queueing</li>
    <li>octavePackages.signal</li>
    <li>octavePackages.sockets</li>
    <li>octavePackages.sparsersb</li>
    <li>octavePackages.splines</li>
    <li>octavePackages.statistics</li>
    <li>octavePackages.stk</li>
    <li>octavePackages.strings</li>
    <li>octavePackages.struct</li>
    <li>octavePackages.symbolic</li>
    <li>octavePackages.tisean</li>
    <li>octavePackages.tsa</li>
    <li>octavePackages.video</li>
    <li>octavePackages.windows</li>
    <li>octavePackages.zeromq</li>
    <li>openems</li>
    <li>openwebrx</li>
    <li>paulstretch</li>
    <li>posterazor</li>
    <li>pothos</li>
    <li>python38Packages.fipy</li>
    <li>python38Packages.python-csxcad</li>
    <li>python38Packages.python-openems</li>
    <li>python38Packages.soapysdr-with-plugins</li>
    <li>python39Packages.fipy</li>
    <li>python39Packages.python-csxcad</li>
    <li>python39Packages.python-openems</li>
    <li>python39Packages.soapysdr-with-plugins</li>
    <li>qradiolink</li>
    <li>rakarrack</li>
    <li>rtl_433</li>
    <li>sdrangel</li>
    <li>seaview</li>
    <li>soapysdr-with-plugins</li>
    <li>solfege</li>
    <li>tigervnc</li>
    <li>urh</li>
    <li>virtualgl</li>
    <li>virtualglLib</li>
    <li>welle-io</li>
    <li>xautoclick</li>
    <li>yoshimi</li>
    <li>zynaddsubfx-fltk</li>
  </ul>
</details>

---

`fldigi` complains during configure:
> ```
> checking for fltk-config... /nix/store/ylfmx4b1hw2q3xmfh7m3qiicrhszp04g-fltk-1.4.x-2021-12-21/bin/fltk-config
> configure: WARNING: fldigi may not work with FLTK 1.4.0
> ```

And the build error:
> ```
>   CXXLD    fldigi
> /nix/store/lbxfixyw1yk099pjyaiy3xj5dl7kxm1g-binutils-2.35.2/bin/ld: /nix/store/ylfmx4b1hw2q3xmfh7m3qiicrhszp04g-fltk-1.4.x-2021-12-21/lib/libfltk.so: undefined reference to `Fl::cairo_make_current(Fl_Window*)'
> collect2: error: ld returned 1 exit status
> ```

Will look into it.

--- a/pkgs/applications/radio/fldigi/default.nix
+++ b/pkgs/applications/radio/fldigi/default.nix
@@ -2,7 +2,7 @@
 , stdenv
 , fetchurl
 , hamlib
-, fltk14
+, fltk13
 , libjpeg
 , libpng
 , portaudio
@@ -31,13 +31,15 @@ stdenv.mkDerivation rec {
     libXinerama
     gettext
     hamlib
-    fltk14
+    fltk13
     libjpeg
     libpng
     portaudio
     libsndfile
     libsamplerate
   ] ++ lib.optionals (stdenv.isLinux) [ libpulseaudio alsa-lib udev ];
+
+  enableParallelBuilding = true;
 
   meta = with lib; {
     description = "Digital modem program";

--- a/pkgs/development/libraries/fltk/1.4.nix
+++ b/pkgs/development/libraries/fltk/1.4.nix
@@ -1,5 +1,5 @@
 import ./common.nix rec {
-  version = "1.4.x-2021-07-04";
-  rev = "1008cdfab27609a6f6a0e82dadad9fd9cbd8a66d";
-  sha256 = "1h057dyhd04b9bjci952b2l7brxv183l9jw9i50mn9qjfljmvqim";
+  version = "1.4.x-2021-12-21";
+  rev = "c8bb2a35850be7c6eaec5ad5a2936a77f7913de2";
+  sha256 = "1fwfg1hp1ajqh2b4ra4phi96854q9i8c0gbyi7pr35ljyv848295";
 }

--- a/pkgs/development/libraries/fltk/default.nix
+++ b/pkgs/development/libraries/fltk/default.nix
@@ -1,5 +1,5 @@
 import ./common.nix rec {
-  version = "1.3.6";
+  version = "1.3.8";
   rev = "release-${version}";
-  sha256 = "0vzk4d6j927v7dxywr5xlqlf70myal1xikkdfvd11p94rcdf9bsv";
+  sha256 = "1pw4ndwn9rr1cxw5qiw32r9la2z9zbjphgsqq1hj2yy4blwv419r";
 }

